### DISCUSSION TITLE
Add a tweak to setup DNS to 1.1.1.1

### DISCUSSION THREAD

**Discussion by CB-Flu** [ChrisTitusTech/winutil#328] [Ideas]
Hi,

Any chance the option to switch DNS to 1.1.1.1 would be added in the Tweaks menu?

Thanks a lot

**Comment by padsalatushal** [comment]
I am currently working on it

**Reply by CB-Flu** [reply]
Thanks a lot

**Reply by toro-** [reply]
Just to further this suggestion...

Maybe add options for filtering DNS like CloudFlare's 1.1.1.2, 1.0.0.2 (malware blocking) and/or 1.1.1.3, 1.0.0.3 (malware and adult content blocking)

Don't know if there are other providers that do this, I am sure there are though.
